packages/grid/x-data-grid/src/hooks/features/clipboard/useGridClipboard.ts

```diff
@@ -26,15 +26,15 @@ function writeToClipboardPolyfill(data: string) {
 }
 
 function hasNativeSelection(element: HTMLInputElement) {
-  if (window.getSelection()?.toString() !== '') {
+  // When getSelection is called on an <iframe> that is not displayed Firefox will return null.
+  if (window.getSelection() && window.getSelection()!.toString() !== '') {
```

Comment:
I can't reproduce what they say in https://developer.mozilla.org/en-US/docs/Web/API/Window/getSelection, it might be even better:

```suggestion
  if (window.getSelection()!.toString() !== '') {
```